app/lib/pdf/finance/invoice_generator.rb

```diff
@@ -79,15 +74,15 @@ def print_logo
 
       def print_address_columns
         # TODO: cleanup the constants
```

Comment:
I think it is about this `7.mm` that remains. It was `97` in the past. Perhaps we can compute this as `FULL_WIDTH - 2*HALF_WIDTH`. So that we have the overall width to match the full width and not hardcode `7` as a constant..

I can give it a try. Together with going over the codeclimate issues that make sense to fix without much refactoring.

Reply:
actually no, I'm removing this comment, what I spoke was about the printer. I did it there anyway.